src/LanguageServer/Impl/LanguageServer.cs

```diff
@@ -111,6 +112,9 @@ private void OnApplyWorkspaceEdit(object sender, ApplyWorkspaceEditEventArgs e)
                 settings.symbolsHierarchyDepthLimit = GetSetting(analysis, "symbolsHierarchyDepthLimit", 10);
                 settings.symbolsHierarchyMaxSymbols = GetSetting(analysis, "symbolsHierarchyMaxSymbols", 1000);
 
+                var linting = pythonSection["linting"];
+                _optionsProvider.Options.LintingEnabled = GetSetting(linting, "enabled", true);
```

Comment:
I don't like using `python.linting.enabled` for our toggle. That's what the extension already uses for toggling pylint/pycodestyle/etc, and having it enabled means prompts and buttons unrelated to the language server. I'd rather use `python.analysis.linting` or something for now.

Reply:
That's a question to @qubitron since multiple almost same settings confuse users. They don't know what analysis is and what is the difference.

Reply:
Seems like the extension now has `python.linting.pylintEnabled` (and so on), so maybe this wouldn't be a big deal. We probably need to look at `python.linting.enabled`, then at something like `python.linting.plsEnabled` or similar to match that pattern.

@DonJayamanne